Mage.Client/src/main/java/org/mage/plugins/card/dl/sources/ScryfallImageSupportCards.java

```diff
@@ -533,6 +533,7 @@ public class ScryfallImageSupportCards {
             add("WOE"); // Wilds of Eldraine
             add("WOT"); // Wilds of Eldraine: Enchanting Tales
             add("WOC"); // Wilds of Eldraine Commander
+            add("CALC"); // Custom Alchemized versions of existing cards. Will download the original in directDownloadLinks
```

Comment:
It's slightly confusing to add a custom set in this list. But if that's the only way for it to work, it's okay.

Reply:
Yeah that first add is for "attempt to download through scryfall", then the directDownloadLinks will direct url download the pillar image from dissension to the correct folder for CALC/1

I did not want to do an heavier rework of the logic, but still have some image on the custom card.